lang/lang_en_fr.txt

```diff
@@ -94,10 +106,22 @@
 "Bed level correct"
 "Corr. niveau plateau"
 
+#MSG_BELTTEST c=17
+"Belt test        "
+"Test de courroie "
+
 #MSG_BED_LEVELING_FAILED_POINT_LOW c=20 r=4
 "Bed leveling failed. Sensor didnt trigger. Debris on nozzle? Waiting for reset."
 "Echec bed leveling. Capt. non declenche. Debris sur buse? En attente d'un reset."
 
+#MSG_BRIGHT c=6
+"Bright"
+"Brill."
```

Comment:
"brill" = lumiere

Reply:
I believe that this "Bright" is not bright, but is a short form of brightness, so a better translation would be `luminosité` but, with the 6 character restriction, we can't use that. In this case, I believe best option is `Brill.`

Reply:
It's for the backlight mode. You have these three options:
Mode Bright -- this is where MSG_BRIGHT is used
Mode Dim
Mode Auto

Reply:
> I believe that this "Bright" is not bright, but is a short form of brightness, so a better translation would be luminosité but, with the 6 character restriction, we can't use that. In this case, I believe the best option is Brill.

"clarté" and "luminosité" it's same  ( sorry for my English)

Reply:
> It's for the backlight mode. You have these three options:
> Mode Bright -- this is where MSG_BRIGHT is used
> Mode Dim
> Mode Auto

it's for oled screen?

Reply:
@carlin57 No, it’s for MK3S LCD screens that have backlight control capability. Prusa started shipping those I think last summer.

Reply:
ok thanks,

Reply:
As this translation represents "Bright", not "Brightness", `Brilliant` is probably the best option(shortened to `Brill.`)

Reply:
> As this translation represents "Bright", not "Brightness", `Brilliant` is probably the best option(shortened to `Brill.`)
Use this.